specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-11-15-preview/managedCassandra.json

```diff
@@ -1345,6 +1538,81 @@
         }
       }
     },
+    "CommandPublicResource": {
+      "description": "resource representing a command",
+      "type": "object",
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "The command which should be run"
+        },
+        "commandId": {
+          "type": "string",
+          "description": "The unique id of command"
+        },
+        "arguments": {
+          "type": "object",
+          "description": "The arguments for the command to be run",
+          "additionalProperties": {
```

Comment:
`additionalProperties` is not allowed. why not use a string array?

Reply:
There are `additionalProperties` used elsewhere in other definition fields. Would those need to be changed in addition to this one to adhere to this standard?